components/product/NotFound.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Rendered when a not found is returned by any of the loaders run on this page
+ */
+
+import { type SectionProps } from "deco/mod.ts";
+import Icon from "deco-sites/persono/components/ui/Icon.tsx";
+import { FnContext } from "deco/types.ts";
+
+interface SubtitleProps {
+  notFoundProductSubtitle: string;
+  notFoundPageSubtitle: string;
+  notFoundPageButtonText: string;
+}
+
+interface Props extends SubtitleProps {
+  /**
+   * @description Adicione $term para no local que o termo buscado devera ser inserido. ex: Nenhum resultado encontrado para "$term".
+   */
+  notFoundProductTitle: string;
+
+  notFoundPageTitle: string;
+}
```

Comment:
voce podia dividir em objetos diferentes para manter uma organização melhor tanto no formulario como no codigo. Alem disso mude a description para ingles, senão o formulario vai ter apenas esse texto em portugues, fica ruim demais assim.
```suggestion
interface Props extends SubtitleProps {
  /**
   * @description Adicione $term para no local que o termo buscado devera ser inserido. ex: Nenhum resultado encontrado para "$term".
   */
  product: { ... };
  page: { ... };
}
```